neuralmonkey/model/sequence.py

```diff
@@ -239,6 +256,7 @@ def __init__(self,
                  add_start_symbol: bool = False,
                  add_end_symbol: bool = False,
                  scale_embeddings_by_depth: bool = False,
+                 embeddings_source: EmbeddedFactorSequence = None,
```

Comment:
Nemela by tady byt jen EmbeddedSequence?

Reply:
Mělo a dokonce tam byl bug že se to nenastavovalo